src/g_editor.c

```diff
@@ -4341,7 +4341,7 @@ void canvas_connect(t_canvas *x, t_floatarg fwhoout, t_floatarg foutno,
 
         /* check if objects are already connected */
     if (canvas_isconnected(x, objsrc, outno, objsink, inno)) {
-        logpost(src, 3, "this pair of iolets is already connected");
+        logpost(src, 3, "io pair already connected");
```

Comment:
this i actually don't like. the term "io" is nowhere used in Pd, and i think it just adds more confusion.

## more suggestions

this one uses the same "cannot connect ..." pattern as the other error-messages in the same code-block:

> cannot connect multiple times

or simply:

> already connected